docs/changelog/94375.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 94375
+summary: Add extension point for `MachineLearning`
+area: Machine Learning
+type: enhancement
```

Comment:
I'm thinking we should make this a `>non-issue`, as this is for internal use. Hopefully if you change the PR labels this file will get deleted, but if it doesn't 10 minutes after changing the labels then please delete it manually.